docs/releases/1.6.0-alpha.1.md

```diff
@@ -0,0 +1,46 @@
+## 1.6.0-alpha.1
+
+1.6.0-alpha.1 is a prerelease early-access of kops 1.6, which is the release with full support for kubernetes 1.6.
+This version of kops & kubernetes has not yet undergone extensive validation, and there will be improvements
+made before release of kops 1.6.0.
+
+This is not a full set of release notes, but rather a summary of the highest impact changes in the 1.6 release:
+
+* kops is not yet recommending etcd3, pending issues such as HA upgrade support.
```

Comment:
This point could use some re-work to explain that etcd3 is new in kubernetes 1.6, but that kops is still defaulting to etcd2 and etcd3 is run-at-your-own-risk.